usecases/no2-monitoring/index.md

```diff
@@ -4,18 +4,52 @@ TROPOMI is the name of a sensor on board of the Sentinel-5 Precursor (S5P) satel
 
 This use case analyses Sentinel 5P imagery, focusing in particular on NO₂ measurements. Compared to other variables measured by TROPOMI, NO₂ is of high interest not only because of its direct relation with environmental health, but also because the main sources are typically known, because it is not transported over long distances and because the total column values measured by TROPOMI are strong indication of the ground level values.
```

Comment:
Similar here, what does TROPOMI stand for?

Reply:
Solved at the beginning of the text.